uptane-standard.md

```diff
@@ -169,13 +169,13 @@ These instructions specify the components necessary for a compliant implementati
 
 The keywords MUST, MUST NOT, REQUIRED, SHALL, SHALL NOT, SHOULD, SHOULD NOT, RECOMMENDED, MAY, and OPTIONAL in this document are to be interpreted as described in {{RFC2119}}.
 
-In order to be considered “Uptane-compliant,” an implementation MUST follow all of these rules as specified in the document.
+In order to be considered "Uptane-compliant", an implementation MUST follow all of these rules as specified in the document.
```

Comment:
Ah yes. I forgot that's one common US-Americanism that I just cannot stand. How about I just drop the quotation marks? I don't think they are necessary anyway. Otherwise I'll revert the comma change.

Reply:
Perfect resolution. Drop the quotation marks.